scripts/common.py

```diff
@@ -24,6 +24,19 @@
 RELEASE_BRANCH_NAME_PREFIX = 'release-'
 
 
+def run_cmd(cmd_str):
```

Comment:
Hang on ... there may have been a miscommunication here. By "keeping it as-is" I meant using a list of command tokens, as opposed to removing the function completely. Sorry for not being clear!

Reply:
Yes, I did not remove the function. I had updated this to use a list of tokens but you mentioned later that it was a nice abstraction and I liked the version of passing string instead of list, so I removed the update. Do you want me to make that update here instead?

Reply:
Ah - to be clear, I was saying that the version of the function that uses list as an input is a nice abstraction, so let's use that please.

Reply:
Yes, got it! Updated now.